src/hb-machinery.hh

```diff
@@ -114,6 +114,7 @@ static inline Type& StructAfter(TObject &X)
 
 #define DEFINE_SIZE_UNBOUNDED(size) \
   DEFINE_INSTANCE_ASSERTION (sizeof (*this) >= (size)) \
+  enum { null_size = (size) }; \
```

Comment:
Dude!  There's a reason this line is not there!

Reply:
I know... the intention was just to have something to start with as you can see from other added comments also